### DISCUSSION TITLE
Trying to find how to rename the URL of a Design view

### DISCUSSION THREAD

**Discussion by madukan** [Budibase/budibase#14455] [General]
Hi, I'm quite new to Budibase, am trying to find how to rename the View URL?
Tried these documentation: 
1. https://docs.budibase.com/docs/screens
2. https://docs.budibase.com/docs/navigation

Need help please

**Comment by mikesealey** [comment]
Hey @madukan - From a screen in the builder, click on the navigation component in the component tree. The settings on the right will allow you to change the label (how it appears in the navigation bar) and the link (the URL) of each screen.

![image](https://github.com/user-attachments/assets/dffa33af-a394-48cd-af10-22ed26a3c400)

**Comment by madukan** [comment]
Hey @mikesealey 
Isn't that link for the navigation? Seem to not allowing me to change (Even if I change it tends to revert back to the drop down values).
Say, in your `/example3` needing to rename to `/example4` - that's what I'm trying to accomplish.
Here's where the root of that path is seen (`/a_test_view_1`): 

<img width="383" alt="image" src="https://github.com/user-attachments/assets/4170fc00-267c-4567-9a29-b7d0a848e4da">

Did I make sense?

**Comment by madukan** [comment]
Okay found one way. Just have to click the white-background area where there aren't any components (just the canvas). 
Then you get the route link, and then updating there can change the URL. However, the navigation doesn't update automatically, so need to go there (your screenshot) and update the URLs manually. 

<img width="544" alt="image" src="https://github.com/user-attachments/assets/5cab235a-6f06-4e3a-b076-6c6d1a192564">
